scripts/start.py

```diff
@@ -106,6 +107,7 @@ def alert_on_exit():
 
 def notify_about_successful_shutdown():
     """Notifies developers that the servers have shutdown gracefully."""
+    extend_index_yaml.main()
```

Comment:
Placing this here.
Initially tried by adding in line 140 (inside callback) (the one in which I accidentally changed the indentation) but it didn't work. Tried a couple more places other than here, but none seems working as expected.

Reply:
Do you know why it didn' work in the callback?

Reply:
Not sure, it gets executed but the problem is it executes in the beginning.

Reply:
Can you try to debug it a bit more? It would be nice to place it around line 140 so that it is clear that this is being executed.